Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-AKS-cluster/sample-workload-for-agic-test.yaml

```diff
@@ -0,0 +1,56 @@
+cat <<EOF | kubectl apply -f -
```

Comment:
What is this file?

Reply:
it is a yaml file to deploy a sample application and ingress in order to test the blue green deployment.
@MattLeach25